packages/lit-element/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [2.4.0] - 2020-08-19
 
 ### Changed
+* Removed build support for TypeScript 3.4.
```

Comment:
These changes should go up into a 3.0 section

Reply:
Done.